src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/Index.cshtml

```diff
@@ -0,0 +1,193 @@
+﻿@using SFA.DAS.Commitments.Support.SubSite.Extensions
+@using SFA.DAS.Commitments.Support.SubSite.Models
+@model ApprenticeshipViewModel
+@{
+    ViewBag.Title = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<div class="grid-row">
+
+    <div class="column-full">
+
+        <h1 class="heading-xlarge">
+            @Model.Name
+        </h1>
+
+        <div>
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Status</h2></th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                    <tr>
+                        <td>Agreement status</td>
+                        <td>@Model.AgreementStatus</td>
+
+                    </tr>
+                    <tr>
+                        <td>Payment status</td>
+                        <td>@Model.PaymentStatus</td>
+
+                    </tr>
+
+                    @if (Model.Alerts != null && Model.Alerts.Any())
+                    {
+                        for (int i = 0; i < Model.Alerts.Count(); i++)
```

Comment:
Instead of using a for loop, we could use a combination of Model.Alerts.First() and foreach(var alert in Model.Alerts.Skip(1)).
Arguably, for loops are more error prone and have the possibility of accessing the array out of bounds, which is not an issue with foreach.